bedrock/styleguide/templates/styleguide/websites/pebbles.html

```diff
@@ -0,0 +1,155 @@
+{% extends 'base-pebbles.html' %}
+
+{% block page_title %}{{ _('Pebbles') }}{% endblock page_title %}
+
+{% block content %}
+<main>
+  <section class="section">
+    <div class="content">
+
+      <h1>Pebbles: Level One Heading</h1>
+
+      <p>Pebbles is the name of our new CSS framework for mozilla.org. This is a very minimal demo page of general markup elements with the base Pebbles styling. This page is <em>not</em> intended as an exhaustive reference of all the various components within Pebbles.</p>
+
+      <p>Lorem ipsum <a href="#link">dolor sit amet</a>, consectetuer adipiscing elit. Donec odio. Quisque <em>volutpat mattis</em> eros. Nullam <strong>malesuada erat</strong> ut turpis. Suspendisse urna nibh, viverra non, <em><strong>semper suscipit</strong></em>, posuere a, pede.</p>
+
+      <h2>Buttons: Level Two Heading</h2>
+
+      <p>We use lots of buttons, both "true" buttons in forms and links visually styled as buttons.</p>
+
+      <p>
+        <button class="button">button</button>
+        <button class="button red">red button</button>
+        <button class="button green">green button</button>
+        <button class="button orange">orange button</button>
+        <button class="button dark">dark button</button>
+      </p>
+
+      <p>
+        <button class="button hollow">hollow button</button>
+        <button class="button hollow red">hollow red button</button>
+        <button class="button hollow dark">hollow dark button</button>
+        <span style="background: gray; padding: 10px 20px; display: inline-block;">
```

Comment:
Yeah... I just wanted to make the button visible, but didn't want to add a class for it, especially since this page has no style sheet of its own. I'll just remove the boxes and some of the buttons will be really hard to see ;)

Of course, this page doesn't *really* need to exist at all, and hopefully won't exist for long because we should start documenting this stuff in the design system.